commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/sftp/SftpFileSystemConfigBuilder.java

```diff
@@ -268,6 +269,21 @@ public ConfigRepository getConfigRepository(final FileSystemOptions opts) {
     public boolean isLoadOpenSSHConfig(final FileSystemOptions opts) {
         return this.getBoolean(opts, LOAD_OPENSSH_CONFIG, Boolean.FALSE);
     }
+    
+    /**
+     * Returns {@link Boolean#TRUE} if exec should be tested and enabled.
```

Comment:
The API returns a `boolean`, not a `Boolean`, so the Javadoc should refer to `true`, not `Boolean#TRUE}.